core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1101,12 +1092,12 @@ public List<CommonUtilsService.ChartsOverviewItem<String, Integer>> selectActivi
       Integer teamId, int numberOfDays, int tenantId) {
     try {
       List<CommonUtilsService.ChartsOverviewItem<String, Integer>> res = new ArrayList<>();
-      List<Integer> fromDb =
+      List<Pair<String, Integer>> fromDb =
           gatherActivityList(
               activityLogRepo.findActivityLogForTeamIdForLastNDays(teamId, tenantId, numberOfDays));
-      for (int elem : fromDb) {
-
-        res.add(CommonUtilsService.ChartsOverviewItem.of("activitycount", elem));
+      for (Pair<String, Integer> elem : fromDb) {
+        log.info("selectActivityLogByTeam {} {}", elem.getKey(), elem.getValue());
```

Comment:
log debug ?

Reply:
Will actually remove that log line completely